client/db/src/bench.rs

```diff
@@ -517,14 +520,37 @@ impl<B: BlockT> StateBackend<HashFor<B>> for BenchmarkingState<B> {
 		self.state.borrow().as_ref().map_or(sp_state_machine::UsageInfo::empty(), |s| s.usage_info())
 	}
 
-	fn proof_size(&self) -> Option<u32> {
+	fn proof_size(&self) -> Option<(u32, u32)> {
```

Comment:
Why do we return here both?

What is the rational behind this? 

If we "compress" always, we are not interested in the uncompressed size? (And we actually should not call this compress, maybe something different)

Reply:
The point was to use this to check the size delta from batch.

At this point we 'compress' never, but if 'proof_size' in bench is only here for PoV, then keeping only one value could be fine.
(I don't know if we want to compress 'always' the parachain could choose, even if there is no real point in not doing it).

So yes I can remove it, just it does not really cost much to keep it.

Reply:
I agree it is relevant to have both the compressed and uncompressed size for analysis.

Reply:
Why is that relevant?